app/presenters/content_change_presenter.rb

```diff
@@ -0,0 +1,45 @@
+class ContentChangePresenter
+  EMAIL_DATE_FORMAT = "%I:%M %P on %-d %B %Y".freeze
+
```

Comment:
Might want to add something here of `delegate :title, :description, :change_note to :content_change`

Or even have this class inherit from [SimpleDelegator](http://ruby-doc.org/stdlib-2.2.1/libdoc/delegate/rdoc/SimpleDelegator.html)

Reply:
Good shout... fixed in b202f42